libs/community/langchain_community/embeddings/huggingface.py

```diff
@@ -167,6 +170,20 @@ def __init__(self, **kwargs: Any):
         except ImportError as e:
             raise ImportError("Dependencies for InstructorEmbedding not found.") from e
 
+        if "show_progress_bar" in self.encode_kwargs:
+            warn_deprecated(
+                since="0.2.5",
```

Comment:
I'm assuming this is the version this would be released in; not sure how to know?

Reply:
Check current version, and then increment patch version by 1